src/undate/undate.py

```diff
@@ -13,7 +13,7 @@
 ONE_DAY = datetime.timedelta(days=1)
 
 
-class Undate:
+class Undate():
```

Comment:
I thought `()` wasn't needed anymore?

Reply:
I am just old school, I guess. We can remove them

Reply:
I am too, trying to train myself not to do this style  `class Undate(Object):` any more